src/page/HomePage.tsx

```diff
@@ -135,9 +164,10 @@ function HomePage(): JSX.Element {
   }, [createTodo, isVisible]);
 
   return (
-    <>
+    <div className="overscroll-x-none overflow-x-hidden">
       <Header />
-      <div className="flex py-2 px-4 items-center justify-center overflow-hidden w-screen h-screen relative">
+      <div className="flex py-2  items-center  overflow-hidden w-screen h-screen relative ">
+        <Sidebar />
         {isLoaded && <Loader />}
         {!isLoaded &&
```

Comment:
use conditional rendering isLoaded -> i means that content already was loaded, rename to `isLoading ? <Loader /> : <Content />`